flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -331,6 +364,16 @@ private Row testDeserialization(
 			.toExternal(deserializedRow);
 	}
 
+	private void testSerDeConsistency(
+		RowData originalRow,
+		CsvRowDataSerializationSchema.Builder serSchemaBuilder,
+		CsvRowDataDeserializationSchema.Builder deserSchemaBuilder) throws Exception {
```

Comment:
Add one more indent for method arguments.

Reply:
fixed.
I pull request for release-1.11 branch:  https://github.com/apache/flink/pull/13444